packages/shared/src/rpc/index.ts

```diff
@@ -150,6 +154,9 @@ const rpcAsset = z.union([
   z.object({ chain: z.literal('Ethereum'), asset: z.literal('ETH') }),
   z.literal('USDC'),
   z.object({ chain: z.literal('Ethereum'), asset: z.literal('USDC') }),
+  z.object({ chain: z.literal('Ethereum'), asset: z.literal('USDT') }),
```

Comment:
oof another one 😕  i think its hard to enforce all assets via type here, can we generate the `{chain,asset}` variants based on our constants?

Reply:
tried it but couldn't make it work right now to generate them :(
problem seems to be with how zod treats arrays

Reply:
okay too bad 😕